### PR TITLE
Add extras.HTML

### PR DESCRIPTION
**Pull request by BeyondEvil**

### CODE REVIEW

File: src/pytest_html/resources/index.jinja2 [hunk 1]
```diff
@@ -41,6 +41,7 @@
         </tr>
         <tr class="extras-row">
           <td class="extra" colspan="4">
+            <div class="extraHTML"> </div>
```
Comment (BeyondEvil) [inline src/pytest_html/resources/index.jinja2]: Should I remove the space?

File: src/pytest_html/resources/index.jinja2 [hunk 2]
```diff
@@ -41,6 +41,7 @@
...
             <div class="media">
               <div class="media-container">
                   <div class="media-container__nav--left"><</div>
```
Comment (BeyondEvil) [inline src/pytest_html/resources/index.jinja2:47]: Is this a typo? Double `<`

File: src/pytest_html/scripts/dom.js [hunk 3]
```diff
@@ -90,6 +90,10 @@ const dom = {
             if (['image', 'video'].includes(format_type)) {
                 media.push({ path: content, name, format_type })
             }
+
+            if (format_type === 'html') {
+              resultBody.querySelector('.extraHTML').insertAdjacentHTML('beforeend', `<div>${content}</div>`)
```
Comment (BeyondEvil) [inline src/pytest_html/scripts/dom.js:95]: The reason I do it this way is that I want the HTML to be in the order it was appended.

File: src/pytest_html/resources/index.jinja2 (hunk 1 above)
Comment (pv-viktor) [reply src/pytest_html/resources/index.jinja2]: pls do

File: src/pytest_html/resources/index.jinja2 (hunk 2 above)
Comment (pv-viktor) [reply src/pytest_html/resources/index.jinja2:47]: Nope, using the lt app & gt char as arrows

**Review by pv-viktor — APPROVED** [review]

**Pull request merged**